frontend/canvas-chrome-ext/src/scripts/compilation-notification.jsx

```diff
@@ -8,6 +8,25 @@ chrome.runtime.onMessage.addListener(function (msg, sender, sendResponse) {
         console.log(msg.data);
         sendResponse("confirmed");
 
-        document.getElementById('update-text').textContent  = JSON.stringify(msg.data, null, '  ');
+        document.getElementById('update-text').textContent  =
+            JSON.stringify(msg.data, null, '  ');
+
+        if(msg.data !== "Your program compiled successfully!") {
+            document.getElementById("comp-status").textContent = "Program Failed to Compile";
+            document.getElementById("res_image").src="img/redx.png";
+        }else{
+            document.getElementById("comp-status").textContent = "Success!";
+            document.getElementById("res_image").src="img/tick.png";
+        }
+    }
+    else if(msg.action === '424error'){
+        console.log(msg);
+        sendResponse("error received from backend");
+
+        document.getElementById('update-text').textContent  =
+            JSON.stringify(msg, null, '  ');
+
+        document.getElementById("comp-status").textContent = "ERROR 424";
```

Comment:
- For submission failures, I'd suggest separating out each line of output from the compilation response into separate `<p>` tags because right now it looks like it's printed out as one long string in the `textContent` attribute, so it's kind of hard to read. 
- Something isn't rendering correctly on my end (the aE character thing): 
- Also regarding the missing `redx.png`, alt text is showing up here instead

![image](https://user-images.githubusercontent.com/75452235/222055362-2b1001dd-e179-4757-8c2a-3a855458ee9b.png)

Reply:
resolving in another PR